### DISCUSSION TITLE
Teoria x Prática - Como contornar essa questão?

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#268] [Discussão]
Fala pessoal, 

1 ano atrás eu escrevi um artigo chamado [Aproximando seus estudos de cenários reais](http://www.felipefialho.com/blog/2015/aproximando-seus-estudos-de-cenarios-reais), que trazia algumas dicas para passar da teoria para a prática.

Essas dicas servem tanto para iniciantes como seniors

E vocês, quais as dicas que podem dar para quem está com essa dificuldade?

**Comment by adrianoavelino** [comment]
Recentemente estou muito encanado no frontend e pra melhorar o meu conhecimentoestou fazendo [freecodecamp](freecodecamp.com), junto várias coisas ao mesmo tempo: estudo de inglês, frontend e tento afastar a procrastinação e no final posso receber um certificado. Nessa mesma onda ainda existe o [#100DaysOfCode](https://twitter.com/hashtag/100DaysOfCode?src=hash) a hashtag já é intuitiva, mas quer entender melhor? [Acesse e conheça esse desafio](https://medium.freecodecamp.com/join-the-100daysofcode-556ddb4579e4#.n2sv2px6s).

**Comment by gabrieldarezzo** [comment]
@adrianoavelino muito louco essa ideia do #100DaysOfCode 
Vou adotar...
